### ISSUE TITLE
Quotes in string

### ISSUE THREAD

**Issue by willm132** [vigneshuvi/SwiftCSVExport#27]
When export the file, in the first row every string has '' quotes around it. Is there anyway we can stop this?

`StringTest` shows as `'StringTest'` in the first row.

**Comment by vigneshuvi** [comment]
@willm132 , Can you please give some example inputs, i will add the solution here.

**Comment by willm132** [comment]
@vigneshuvi 
let a = "StringTest1"
let b = "StringTest2"
let user1:NSMutableDictionary = NSMutableDictionary()
user1.setObject(a, forKey: "Column1" as NSCopying);
user1.setObject(b, forKey: "Column2" as NSCopying);
data.add(user1);

This prints out in csv as

'StringTest1' StringTest2
As you can see the first row has '' around them while the second does not

**Comment by vigneshuvi** [comment]
@willm132 ,

### Issue is resolved in version SwiftCSVExport (2.6.0)

Example:
```let a = "StringTest1"
let b = "StringTest2"
let auser1:NSMutableDictionary = NSMutableDictionary()
auser1.setValue(a, forKey: "Column1");
auser1.setValue(b, forKey: "Column2");
        
let data:NSMutableArray  = NSMutableArray()
data.add(auser1);
        
let header = ["Column1", "Column2"]
// Create a object for write CSV
let writeCSVObj = CSV()
writeCSVObj.rows = data
writeCSVObj.delimiter = DividerType.comma.rawValue
writeCSVObj.fields = header as NSArray
writeCSVObj.name = "userlist"
        
// Enable Strict Validation
CSVExport.export.enableStrictValidation = true

let output = CSVExport.export(writeCSVObj);
XCTAssertEqual(true, output.result.isSuccess)
if output.result.isSuccess {
     guard let filePath =  output.filePath else {
           print("Export Error: \(String(describing: output.message))")
           return
      }
      self.testWithFilePath(filePath, rowCount: data.count, columnCount: header.count)
     print("FilePath: \(filePath)")
} else {
            print("Export Error: \(String(describing: output.message))")
}
```

**Comment by willm132** [comment]
@vigneshuvi issue has been resolved. Thank you
